hexa/metadata/graphql/schema.graphql

```diff
@@ -0,0 +1,107 @@
+"""
+Interface for type implementing metadata
+"""
+interface HasMetadata {
+  extendedId: String!   # Encoded concatenation of the class model type and the instance id
+  metadata: MetadataObject!
+}
+"""
+Generic metadata attribute
+"""
+type MetadataAttribute {
```

Comment:
Since we have deleted the object between the business object and the attributes, you can add a 
`target/object: MetadataObject` here